full/Dockerfile

```diff
@@ -70,7 +70,7 @@ RUN $HOME/.cargo/bin/rustup update \
     && $HOME/.cargo/bin/rustup component add rls-preview rust-analysis rust-src
 
 # Install nvm with node and npm
-ENV NODE_VERSION 8
+ENV NODE_VERSION 10
```

Comment:
Working on Theia requires node version 8. We'd need to create a custom image for Theia then, right?

Reply:
It's true that upgrading to node 10 used to break Theia's install, but this has now been fixed. Are there any other Theia / node 10 blockers that I'm not aware of? We should probably test this, or maybe take it upstream.

Reply:
That's a good question. I've seen a lot of effort towards node 10 support, but I'm not sure how far this is.

I just did a quick test, and there is still [something to to on the Theia side](https://github.com/theia-ide/theia/issues/2009#issuecomment-446205981).

Reply:
+1 for bumping. We can handle it in Theia's `.gitpod.yml`